src/bootstrap/check.rs

```diff
@@ -66,7 +66,7 @@ impl Step for Rustc {
         });
     }
 
-    /// Build the compiler.
+    /// Builds the compiler.
```

Comment:
Why is this an improvement rather than a random stylistic change?
What will prevent people from using their preferred form again?

=> controversial

Reply:
Not controversial at all. This is the standard style, to use the present tense. What prevents this happening again is people like you and me making sure this is used when reviewing PRs.

Reply:
"Build the compiler" is also present tense, so that's not even the point here. The difference is about using 2nd or 3rd person singular.

But indeed the majority of doc comments seem to be written in 3rd person.

Reply:
@RalfJung 
[ot] I never realized imperative is actually a usual present tense with implicit "you" in English, it's an entirely separate form without a tense in Russian.

Reply:
Yeah same in Germany... English just has the most boring verbs.^^

I guess it's actually meant to be imperative here, not 2nd person singular. Whatever.

Reply:
@RalfJung @petrochenkov Well yes, but it's the imperative mood... the issue is so many verb forms are degenerate in English. In any case, the convention seems to be 3rd person singular present indicative. ;-) I.e., "[This function] builds the compiler." This makes sense to me.